templates/submit_pdf_array.R

```diff
@@ -0,0 +1,38 @@
+#------------------------------------------------------
+# AUTHOR: Emily Linebarger 
+# PURPOSE: Example script for submitting PDF array jobs 
+# LAST UPDATED: June 16, 2020 
+#------------------------------------------------------
+rm(list=ls())
+
+# You will need to be in the root of the 'ihme.covid' repository. 
+repo_root = "~/repos/covid19/ihme.covid/"
```

Comment:
I need to modify this to download `ihme.covid` to a temp directory - but I wasn't sure how it would source these template files given that they're not functions.

Reply:
I guess to be clear - it would be easy to modify this to download the repo to a temp folder. But in order to get these templates, my understanding is that the user would still have to clone the repository. So I'm trying to avoid this duplication.

Reply:
There's a way, I just have to go look it up.

Do you want to **source** the files from inside another R script or just **run** the files as stand-alone R scripts?

Reply:
Just run as standalone scripts - I'm thinking more and more these scripts don't need to live in this repository.